e2e/src/usecases/OffRamps.js

```diff
@@ -60,28 +63,6 @@ export default offRamps = () => {
       await element(by.id('cashOut')).tap()
     })
 
-    it.each`
```

Comment:
this test is same as the test below except for using different amounts and asserting for SendBar.